7-React/src/components/TodoItem/TodoItem.spec.tsx

```diff
@@ -0,0 +1,48 @@
+import TodoItemDriver from "./TodoItem.driver";
+import { Chance } from "chance";
+
+describe("Todo Item ", () => {
+  let driver: TodoItemDriver;
+  const chance = Chance();
```

Comment:
you can initialize it outside the describe, just in case other describes want to use it